aws/cerberus/cerberus.tf

```diff
@@ -358,16 +372,16 @@ resource "aws_lb" "cerberus_alb" {
 resource "aws_lb_target_group" "cerberus_alb_target_group" {
   count = "${var.create && var.enable_https ? 1 : 0 }"
 
-  name                 = "${local.module_prefix}-https"
-  port                 = "443"
-  protocol             = "HTTPS"
+  name                 = "${local.module_prefix}-${lower(local.alb_protocol)}"
+  port                 = "${var.alb_target_group_port}"
+  protocol             = "${local.alb_protocol}"
   vpc_id               = "${data.terraform_remote_state.vpc.vpc_id}"
   target_type          = "instance"
   deregistration_delay = "300"
 
   health_check {
     path                = "/login"
-    protocol            = "HTTPS"
+    protocol            = "${local.alb_protocol}"
```

Comment:
Given the ALB protocol and the target group protocol could differ (HTTPS vs HTTP), you may want to use the variable suggested above `alb_target_group_protocol`.